mobile/build.gradle

```diff
@@ -12,12 +12,12 @@ ext {
 }
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 29
```

Comment:
Why downgrade the SDK to `29`?

Reply:
To show other modules in AS. They were not showing without this change. Can be abandoned.